### DISCUSSION TITLE
Kubernetes client - Patch

### DISCUSSION THREAD

**Discussion by bonfi7** [dotnet/dotnet-operator-sdk#767] [Q&A]
Hi, I’ve started using the client and notice that  save method use a combination of create and update. I wonder if it is possibile instead using patching or expose another generic method for patching. What do you think about it?

**Comment by gabrieledemaglie** [comment]
Hi @bonfi7,

it would be nice to have a patch method indeed, so we can have the same behavior of the "kubectl apply" command.

**Reply by buehler** [reply]
Hey @bonfi7 and @gabrieledemaglie 

This is already discussed in an issue:
https://github.com/buehler/dotnet-operator-sdk/issues/559

**Comment by gabrieledemaglie** [comment]
Hi @buehler , you are right! :)

I'll follow that issue!
